src/pages/resultsView/querySummary/QuerySummary.tsx

```diff
@@ -136,9 +136,11 @@ export default class QuerySummary extends React.Component<{ routingStore:Extende
     }
 
     @autobind
-    onSubmit(){
-        this.closeQueryForm();
-        getGAInstance()('send', 'event', 'resultsView', 'query modified');
+    onSubmit(submitSucceeded:boolean){
```

Comment:
not important but would it make more sense not to call onSubmit in case of failure?